template/.github/workflows/{% if AddLinkCheck %}link-check.yml{% endif %}

```diff
@@ -0,0 +1,33 @@
+name: link-check
+
+on:
+  push:
+    branches:
+    - main
+  pull_request:
+    branches:
+    - main
+  schedule:
+    - cron: '0 0 * * 0' # every Sunday at 00:00
+
+jobs:
+  linkChecker:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+
+      - name: Link Checker
+        id: lychee
+        uses: lycheeverse/lychee-action@v1
+        with:
+          # fail action if there are broken links
+          fail: true
```

Comment:
Any links to http://localhost other private addresses will fail. For example running web server for docs locally. These addresses can be excluded using `--exclude-all-private` argument.

 ```suggestion
           fail: true
           args: "--exclude-all-private"
```